### PR TITLE
[JBPM-9946] servicesorm.xml validation failure

### PR DESCRIPTION
**Pull request by fjtirado**
As proposed by @martinweiler  in JIRA description

**JIRA**:

[link](https://issues.redhat.com/browse/JBPM-9946)

### CODE REVIEW

File: jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<entity-mappings xmlns="http://java.sun.com/xml/ns/persistence/orm"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_0.xsd"
-  version="2.0">
+<entity-mappings xmlns=http://xmlns.jcp.org/xml/ns/persistence/orm
+  xmlns:xsi=http://www.w3.org/2001/XMLSchema-instance
+  xsi:schemaLocation=http://xmlns.jcp.org/xml/ns/persistence/orm orm_2_1.xsd
+ version="2.1">
```
Comment (martinweiler) [inline jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml:6]: Can you restore the quotes? Without them, I get a validation error on https://www.freeformatter.com/xml-validator-xsd.html

File: jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml [hunk 2]
```diff
@@ -1383,76 +1107,351 @@
...
         SELECT processInstanceId, processId, processName, processVersion, status, externalId, start_date, user_identity, correlationKey 
           FROM ProcessInstanceLog pil 
          WHERE processInstanceId IN (:idList) 
     </query>
   </named-native-query>
-  
+
   <named-native-query name="GetVariablesByProcessInstanceIdList">
       <query>
             SELECT A1.processInstanceId, A1.variableId, A1.value
```
Comment (martinweiler) [inline jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml:1457]: Maybe the line numbers in RHPAM-4649 were incorrect (my bad, I should have double-checked), but the correct sequence order as per the xsd is named-query, named-native-query, sql-result-set-mapping. Therefore, please move <sql-result-set-mapping name="UserTaskInstanceDesc"> to the end of the file (after the last </named-native-query>). Thanks!

**Review by martinweiler — CHANGES_REQUESTED** [review]

**Review by gmunozfe — APPROVED** [review]
Looks good to me, excellent work @fjtirado !

**Review by martinweiler — APPROVED** [review]
LGTM, thank you @fjtirado

**Pull request merged**